tests/integration/tests/poi.js

```diff
@@ -245,7 +245,7 @@ afterEach(async () => {
   try {
     await clearStore(page) /* if only the above test is run page is not used */
   } catch (e) {
-    console.error(e)
+    fire('error_h', `poi clear store error ${e}`)
```

Comment:
why this change ? Is `error_h` still used ?

Reply:
it come from replaceAll before a build another error handler